src/constants.ts

```diff
@@ -17,10 +17,25 @@ export const SAVES_ZIP_PREFIX = "poketernity_";
 export const SAVE_FILE_EXTENSION = "txt";
 
 /** Prefix for local storage keys. */
-export const LS_PREFIX = "pkty";
+export const LS_PREFIX = "pky";
```

Comment:
```suggestion
export const LS_PREFIX = "pkty";
```
adding it since i made other suggestions so you don't need a separate commit